README.md

```diff
@@ -586,6 +589,153 @@ onMounted(() => {
 
 ---
 
+## Path
+
+Get or set the path parameter from the [MQTT Options](#mqtt-options).
+
+### Type Definition
+
+```ts
+const path: (e?: string) => string;
+```
+
+### Get Path
+
+```ts
+$mqtt.path(); // ie: "/mqtt"
+```
+
+### Set MQTT path
+
+```ts
+$mqtt.path('/ws/mqtt');
+```
+
+### Example usage
+
+```html
+<template>
+  <label>MQTT path: {{ $mqtt.path() }}</label>
+</template>
+```
+
+```ts
+onMounted(() => {
+  console.log(this.$mqtt.path());
+});
+```
+
+### Composition API
+
+```ts
+import { onMounted } from 'vue';
+import { $mqtt } from 'vue-paho-mqtt';
+
+onMounted(() => {
+  console.log($mqtt.path());
+});
+```
+
+---
+
+## Username
+
+Get or set the username parameter from the [MQTT Options](#mqtt-options).
+
+### Type Definition
+
+```ts
+const username: (e?: string) => string;
+```
+
+### Get Username
+
+```ts
+$mqtt.username(); // ie: ""
+```
+
+### Set MQTT username
+
+```ts
+$mqtt.username('testUsername');
+```
+
+### Example usage
+
+```html
+<template>
+  <label>MQTT username: {{ $mqtt.username() }}</label>
+</template>
+```
+
+```ts
+onMounted(() => {
+  console.log(this.$mqtt.username());
+});
+```
+
+### Composition API
+
+```ts
+import { onMounted } from 'vue';
+import { $mqtt } from 'vue-paho-mqtt';
+
+onMounted(() => {
+  console.log($mqtt.username());
+});
+```
+
+---
+
+## Password
+
+Get or set the password parameter from the [MQTT Options](#mqtt-options).
+
+### Type Definition
+
+```ts
+const password: (e?: string) => string;
+```
+
+### Get Password
```

Comment:
You can add a disclaimer warning here since exposing the password to the client can cause security issues if not used properly.

Reply:
Add disclaimer with caution red color for password